eng/common/TestResources/New-TestResources.ps1

```diff
@@ -359,9 +369,10 @@ try {
         $ProvisionerApplicationOid = $sp.Id
     }
 
-    $tags = @{
-        Owners = (GetUserName)
-        ServiceDirectory = $ServiceDirectory
+    $tags = @{ Owners = (GetUserName) }
+
+    if ($ServiceDirectory) {
```

Comment:
Should we include any other info?

Reply:
I'm not sure.   Maybe adding a $Product analog to $ServiceDirectory would be useful.  For resource group naming, etc